copter/source/docs/airspeed-estimation.rst

```diff
@@ -1,9 +1,102 @@
+4. Switch to :ref:`AltHold <altholdmode>` and let the vehicle drift downwind to establish the wind direction and speed
+5. Yaw the vehicle so the nose faces into the wind and apply full forward pitch until the vehicle has finished accelerating upwind
+6. Release the pitch stick and allow the vehicle to decelerate and then drift back to the starting line
+7. Reposition the vehicle to a comfortable position
+8. Yaw the vehicle 90 deg (e.g. clockwise) to the wind and apply full left roll stick until the quad has finished accelerating.  Release the stick and allow the vehicle to again decelerate and drift back downwind to the starting line
+9. Reposition the vehicle to a comfortable position
+10. Repeat step 8 but with the vehicle facing backwards into the wind (pilot should pull back on the pitch stick to accelerate into the wind)
+11. Repeat step 8 but with the vehicle's right side facing into the wind (pilot should apply full right roll to accelerate into the wind)
+12. Download the :ref:`onboard logs <common-downloading-and-analyzing-data-logs-in-mission-planner>` for analysis as described in the video
+
+The final value for :ref:`EK3_DRAG_MCOEF <EK3_DRAG_MCOEF>` is normally between 1.1 and 2.0.
+
+Viewing Windspeed and Direction in Real-Time
+--------------------------------------------
+
+Once enabled, the estimated wind speed is sent to the ground station in real-time via the `WIND <https://mavlink.io/en/messages/ardupilotmega.html#WIND>`__ mavlink message.  If using Mission Planner, on the Data page, select the "Quick" tab, double-click on any number shown and select "wind_dir" or "wind_vel".
+
+.. image:: ../images/windspeed-mp.png
+    :target: ../_images/windspeed-mp.png
+    :width: 450px
+
+Barometer Position Error Compensation
+-------------------------------------
+
+Once wind estimation has been enabled, it can be used to compensate for the wind's effect on each barometer in each of four directions (foward, back, left right).  The method to calculate each parameter's value is discussed `here in the video <https://youtu.be/xVVtvVuZGQE?t=3486>`__.
+
+Perform a test flight
+
+- Fly the vehicle nose-first into the wind, then tail-first, then right-side first, then left-side first
+- Land and download the :ref:`onboard logs <common-downloading-and-analyzing-data-logs-in-mission-planner>`
+
+Calculate the position error coefficient for each direction
+
+- Calculate the air density (in kg/m^3) using an online calculator (`like this one <https://www.density.co.uk/calculators/density-of-air>`__)
+- Calculate the static pressure error (in Pa) = -height * gravity * air density
+- Calculate the dynamic pressure = 0.5 * air density * airspeed^2
+- Position error coefficient = static pressure error / dynamic pressure.  This value should be between -1 and +1
+
+Enable the compensation and copy the above calculated position error coefficients into the appropriate parameters
+
```

Comment:
its a matter of style, but I have been trying to not link stuff like BAROx_ ...but rather say:
"for example for the first barometer: 
BARO1_...etc. "
so that the param label matches where you wind up...

Reply:
Txs for the review, changed.